lib/rbs/trace/inline_comment_visitor.rb

```diff
@@ -58,6 +58,17 @@ def find_method_definition(name)
           member.is_a?(AST::Members::MethodDefinition) && member.name == name
         end
       end
+
+      def with_context(node)
+        names = node.constant_path.full_name_parts
+        @context.push(*names)
+
+        yield
+
+        names.each do
+          @context.pop
+        end
```

Comment:
What about passing arguments?

```suggestion
        @context.pop(names.size)
```